tests/generate_packageTest.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
@Adrianzatreanu you sure created a unit test for running main() ? I can't see it, maybe you forgot to commit it...